### ISSUE TITLE
`Wally search` should be useable without `wally.toml` in directory

### ISSUE THREAD

**Issue by Barocena** [UpliftGames/wally#139]
currently `Wally search` command does not work if the terminal is not in project directory

it would be a cool QoL change if it is possible

**Comment by u-train** [comment]
my 2c:

It's a technical reason, but to search, Wally needs to know what registry to search in. Right now, your manifest file would specify that, hence the dependence.

But, we can assume that people will be searching in Uplift's registry 99% of the time. Could also provide an optional argument in the rare case someone wants to search in some alternative registry.

Though, I have to actually wonder how often someone would need to use wally search outside a valid project. Especially because [there is a website to search for packages](https://wally.run). This feature is small, but is it even worth adding?

**Comment by magnalite** [comment]
Yeah I'm really unsure what to do about this one. In general we should avoid any assumptions about which registry the user wants to use, this is why this currently doesn't work. Additionally, as u-train pointed out, there is also the website which provides an easy way to search the pubic registry already.

That said, if a blind search is asked for then it is exceedingly likely the user just wants to search the public registry so this would be a nice and easy QoL change. We can bench this for later but I think we should decide on a final outcome before we hit 1.0.0.
